extras/speaker-recognition/src/simple_speaker_recognition/services/transcription/deepgram.py

```
@@ -0,0 +1,254 @@
+
+    async def health_check(self) -> bool:
+        """Check Deepgram API accessibility."""
+        try:
+            async with aiohttp.ClientSession() as session:
+                async with session.get(
+                    f"{self.base_url}/v1/projects",
+                    headers={"Authorization": f"Token {self.api_key}"},
+                    timeout=aiohttp.ClientTimeout(total=5),
+                ) as response:
+                    return response.status == 200
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)